src/main/java/adris/altoclef/tasks/slot/EnsureFreeCraftingGridTask.java

```diff
@@ -0,0 +1,47 @@
+package adris.altoclef.tasks.slot;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.helpers.StorageHelper;
+import adris.altoclef.util.slots.PlayerSlot;
+import adris.altoclef.util.slots.Slot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.resource.ResourceManager;
+
+public class EnsureFreeCraftingGridTask extends Task {
```

Comment:
If it's just for the 2x2 grid maybe call it `Clear2x2CraftingGridTask` or something

Reply:
done!